src/component/feature/FeatureView/FeatureVariants/FeatureVariantsList/AddFeatureVariant/useOverrides.ts

```diff
@@ -0,0 +1,41 @@
+import { useReducer } from 'react';
+import { IOverride } from 'interfaces/featureToggle';
+
+type OverridesReducerAction =
+    | { type: 'SET'; payload: IOverride[] }
+    | { type: 'CLEAR' }
+    | { type: 'ADD'; payload: IOverride }
+    | { type: 'REMOVE'; payload: number }
+    | { type: 'UPDATE_VALUES_AT'; payload: [number, IOverride['values']] }
+    | { type: 'UPDATE_TYPE_AT'; payload: [number, IOverride['contextName']] };
+
+const overridesReducer = (
```

Comment:
Would like to see a test for this reducer, since that is one of the great benefits of using this pattern.